### PR TITLE
Edit comments

### PR DESCRIPTION
**Pull request by guillermooo**
Revise based on recent changes to the symbol def topic.

### CODE REVIEW

**Comment by guillermooo** [comment]
@FichteFoll PTAL

File: source/reference/comments.rst [hunk 1]
```diff
@@ -174,19 +185,18 @@ Top-level Elements
...
-   Defines an end marker for a comment block.
+   Used to define an end marker
+   for block comments (use ``TM_COMMENT_END_2``).
 
-   If omitted,
-   ``TM_COMMENT_START`` will be treated as a line comment marker.
-
-   To add more types of comment end markers,
-   use a name like ``TM_COMMENT_END_2``.
+   .. XXX not sure about the following
```
Comment (FichteFoll) [inline source/reference/comments.rst]: Why so? That's pretty much how it works. Refer to lines 43-48 in `Default/comments.py`.

Edit: Actually, this entire section with TM_COMMENT_START and _END is kinda wrong now. You should read into `comments.py`.

And btw, the `TM_COMMENT_DISABLE_INDENT` example block is wrong. It should also be mentioned that this is expected to be `yes` or `no` (strings).

Comment (guillermooo) [inline source/reference/comments.rst]: > And btw, the TM_COMMENT_DISABLE_INDENT example block is wrong. It should also be mentioned that this is expected to be yes or no (strings).

Fixed.

> Edit: Actually, this entire section with TM_COMMENT_START and _END is kinda wrong now. You should read into comments.py.

Will check, but I don't think we should explain implementation details. I think the point is that `TM_COMMENT_START` should be used for line comments, and `TM_COMMENT_START[END]_2` should be used for block comments.

I don't think ST uses the comment.py code for anything else?

Comment (FichteFoll) [inline source/reference/comments.rst]: Well, ST would still interpret it correctly, but how would you add two line marker comments like for INI files (`;` and `#`)? Or maybe there are languages with more than one block comment marker (Lua has infinite for example).

Technically, the previous description was more correct and didn't give off "too much information" either.

> I don't think ST uses the comment.py code for anything else?

I don't think I understand what you mean with this.

**Pull request merged**